scripts/modules/createFeed.mjs

```diff
@@ -15,14 +15,15 @@ export default function createFeed(feed, counter){
 
   if(index === 0){
     feedContainer.innerHTML = ""
-  };  
+  };
+  if(loaded > feed.length){loaded = feed.length}  
 
   for(index; index < loaded; index++){
     let image = defaultImage;
     const time = feed[index].created.slice(11,19) + ", " + feed[index].created.slice(0,10);
 
-    if(feed[index].author.avatar.length > 1){
```

Comment:
This is a fix to accommodate the api changes, previously it returned an empty string if the user had no avatar but now it return "null" so it you're testing on the old version on the api this will cause an issue.